include/votca/xtp/gw.h

```diff
@@ -63,6 +63,9 @@ class GW {
     double qp_grid_spacing;  // Spacing of grid points in Ha
     Index gw_mixing_order;   // mixing order
     double gw_mixing_alpha;  //  mixing alpha, also linear mixing
+    std::string quadrature_scheme;  // Kind of Gaussian-quadrature scheme to use
+    Index order = 12;  // only needed for complex integration sigma default:12
+    double alpha = 0.001;
```

Comment:
what is alpha?

Reply:
https://aip.scitation.org/doi/10.1063/1.5144277
Equation 28 and 29

Reply:
And unsued?

Reply:
what?

Reply:
If you meant unused, you are right. It is something I inherited from a different implementation and doing some tests with the old implementation it seems not so important after all. I will remove it everywhere.